tests/run_examples.py

```diff
@@ -165,6 +165,7 @@ def check_library(rule):
 req_library["basic_statistics_spmd.py"] = ["dpctl", "mpi4py"]
 req_library["covariance_spmd.py"] = ["dpctl", "mpi4py"]
 req_library["dbscan_spmd.py"] = ["dpctl", "mpi4py"]
+req_library["dbscan_array_api.py"] = ["array_api_strict"]
```

Comment:
Shouldn't this one also be added to the test requirements, since these example files are also executed as part of tests?

Reply:
It is already in testing env